Moved issue's comment 'Select text block...' position

sanak opened the pull request:
Signed-off-by: Ko Nagase <nagase@georepublic.co.jp>

Supports #13.

Changes proposed in this pull request:
- Moved issue's comment 'Select text block...' position from after buttons to before buttons.
![text_block_in_wide_range_new](https://user-images.githubusercontent.com/629923/124241537-cd1f8f00-db56-11eb-8ef9-7805af0e8b13.png)
![text_block_in_narrow_range_new](https://user-images.githubusercontent.com/629923/124241565-d27cd980-db56-11eb-8819-4f0fec2557c9.png)


@gtt-project/maintainer

sanak commented:
@mopinfish, @dkastl  
Well, I will try to find dynamic responsive solution (move template position only in narrow browser width (or mobile OS) case) to avoid to confuse existence users, so please wait for a while.

dkastl commented:
I wouldn't spend so much time on this. I think it's better for all screens to show the text template dropdown first.

So just announce it as an improvement, that's enought! There is no need to excuse improvements ;-)

sanak commented:
> I wouldn't spend so much time on this. I think it's better for all screens to show the text template dropdown first.
> 
> So just announce it as an improvement, that's enought! There is no need to excuse improvements ;-)

@dkastl Oh, yes. I will not spend too much time about this, but actually, I felt that too much emphasizing the plugin's control is not good, so I will try responsive way a bit. 😅

sanak commented:
Well, I will try MediaQuery + `display: inline-flex` with `order` from referring the following Japanese articles.
- https://shu-naka-blog.com/css/order/
- https://sole-color-blog.com/blog/71/

sanak commented:
@mopinfish (CC: @dkastl)  
I reverted original commit and added responsive (media query) with `display: inline-flex` with order, so could you check the diff again ?  
Thanks,

mopinfish commented:
@sanak changes looks good. But, I am worried about  failed test for v4.1.

Don't you have to worry about this?

sanak commented:
@mopinfish Oh, I missed it. Thanks for mentioning about it.
The error says `LoadError: cannot load such file -- /var/test/test_helper`, so I guess that some setup is missing, but I will check it a bit, later.

sanak commented:
@mopinfish (CC: @dkastl)  
Hmm... About the test error, I confirmed that same query was no problem on my local macOS environment, so there may be current using GitHub actions itself issue...
```
% RAILS_ENV=test bundle exec rake --trace redmine:plugins:test NAME=redmine_text_blocks
Unknown database adapter `postgis` found in config/database.yml, use Gemfile.local to load your own database gems
** Invoke redmine:plugins:test (first_time)
** Execute redmine:plugins:test
** Invoke redmine:plugins:test:units (first_time)
** Invoke db:test:prepare (first_time)
** Invoke db:load_config (first_time)
** Invoke environment (first_time)
** Execute environment
** Execute db:load_config
** Execute db:test:prepare
** Invoke db:test:load (first_time)
** Invoke db:test:purge (first_time)
** Invoke db:load_config 
** Invoke db:check_protected_environments (first_time)
** Invoke db:load_config 
** Execute db:check_protected_environments
** Execute db:test:purge
** Execute db:test:load
** Invoke db:test:load_schema (first_time)
** Invoke db:test:purge 
** Execute db:test:load_schema
** Execute redmine:plugins:test:units
Mocha deprecation warning at /Users/sanak/Projects/gtt/git/redmine/vendor/bundle/ruby/2.6.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require': Require 'mocha/test_unit', 'mocha/minitest' or 'mocha/api' instead of 'mocha/setup'.
** Invoke redmine:plugins:test:functionals (first_time)
** Invoke db:test:prepare 
** Execute redmine:plugins:test:functionals
** Invoke redmine:plugins:test:integration (first_time)
** Invoke db:test:prepare 
** Execute redmine:plugins:test:integration
Run options: --seed 38448

# Running:

........

Finished in 1.242922s, 6.4364 runs/s, 49.8825 assertions/s.
8 runs, 62 assertions, 0 failures, 0 errors, 0 skips
```

I will try dummy PR for check which is the cause of this issue.

sanak commented:
@mopinfish (CC: @dkastl)  
The following dummy PR (only change `README.md`) CI tests failed, so I think that CI tests failure is another problem.
https://github.com/gtt-project/redmine_text_blocks/pull/15

Are you okay to merge this once, then create another issue for CI test failure ?

mopinfish commented:
@sanak 
I got about CI Problem.
So , I think this issue is LGTM.

sanak commented:
@mopinfish Okay, thanks for confirmation. Then I will merge this with squash mode.
